cms/templates/js/move-xblock-list.underscore

```diff
@@ -35,6 +31,17 @@
                     <span class="icon fa fa-arrow-right forward-sr-icon" aria-hidden="true"></span>
                     <span class="sr forward-sr-text"><%- gettext("Click for children") %></span>
                 </button>
+            <% } else { %>
+                <span class="component">
+                    <span class="xblock-displayname truncate">
+                        <%- xblock.get('display_name') %>
+                    </span>
+                    <% if(currentLocationIndex === i) { %>
+                        <span class="current-location">
+                            (<%- gettext('Current location') %>)
```

Comment:
@muzaffaryousaf I can take a look; I'll be finishing up this Move feature for doc.